411-dmz-example-1/azuredeploy.json

```diff
@@ -0,0 +1,521 @@
+﻿{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
```

Comment:
Removed parameter and added a new variable:
"StorageAccountName": "[concat(uniquestring(resourceGroup().id), 'dmz1')]",